packages/selection/frontend/sirius-components-selection/src/SelectionDialog.tsx

```diff
@@ -162,19 +151,7 @@ export const SelectionDialog = ({
                 data-testid={selectionObject.label}>
                 <ListItemIcon>
                   {selectionObject.iconURL.length > 0 ? (
-                    <div className={classes.iconContainer}>
-                      {selectionObject.iconURL.map((icon, index) => (
-                        <img
-                          height="24"
-                          width="24"
```

Comment:
This is changing the icon size from 24×24 to 16×16.

Reply:
I added the possibility to custom icon size.
Done